editor/src/core/shared/dom-utils.ts

```diff
@@ -274,3 +275,13 @@ export function getDomRectCenter(rect: DOMRect): { x: number; y: number } {
     y: rect.y + rect.height / 2,
   }
 }
+
+export function defaultDisplayTypeForHTMLElement(elementName: string): 'inline' | 'block' | null {
```

Comment:
hmmm, this only works if the user doesn't overwrite the defaults with a css. 
I _think_ in most/all places where we store browser defaults, we only use them to inform visual indicators, and not to make decisions based on them, but maybe I am totally wrong about this

Reply:
This prevents setting `display: block` on divs with the autoconversion dragging

Reply:
could you leave a comment here please to indicate that this will need to be smarter if the user has global css overrides?